### PR TITLE
feat: send_message_ui  #31

### PR DESCRIPTION
**Pull request by gagan-aryan**

### CODE REVIEW

**Comment by gagan-aryan** [comment]
issue #31

File: client/src/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import ReactDOM from 'react-dom';
 import './index.css';
 
 import ChatLink from './pages/chatlink';
-import Messaging from './pages/messaging';
+import Messaging from './pages/messaging/main';
```
Comment (muke1908) [inline client/src/index.js:6]: Any specific thought behind putting it inside `main` dir ?

File: client/src/pages/messaging/main/index.js [hunk 2]
```diff
@@ -78,22 +80,72 @@ const Chat = () => {
         ))}
       </div>
       <br />
-      <div>
-        {messages.map(({ body, sender }, i) => (
-          <div key={i}>
-            <b>{sender === userId ? 'You: ' : 'They: '}</b>
-            {body}
+      <div
```
Comment (muke1908) [inline client/src/pages/messaging/main/index.js:83]: Looks like you forgot to replace with css classes

Comment (gagan-aryan) [reply client/src/pages/messaging/main/index.js:83]: No. The changes were not getting reflected in the ui

File: client/src/index.js (hunk 1 above)
Comment (gagan-aryan) [reply client/src/index.js:6]: index.js and Style.module.css should have done ig. Was just separating them into a new folder as well

**Comment by muke1908** [comment]
- Can you remove all those public key info for the page? But keep the user information at the top. (Check #31 )
 - Inline css is bad practice.

**Comment by gagan-aryan** [comment]
Inline css shall never be used again.

**Pull request merged**